oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/importer/IndexImporter.java

```diff
@@ -76,6 +78,10 @@
     private final IndexDefinitionUpdater indexDefinitionUpdater;
     private final boolean preserveCheckpoint = Boolean.getBoolean(OAK_INDEX_IMPORTER_PRESERVE_CHECKPOINT);
 
+    static final int RETRIES = Integer.parseInt(System.getProperty("oak.index.import.retries", "5"));
```

Comment:
FYI you could also use:

    Integer.getInteger("oak.index.import.retries", 5);

Reply:
Done